en/micro-integrator/docs/references/config-catalog.md

```diff
@@ -1229,15 +1229,630 @@ retry_attempts = ''
 </div>
 
 
+## Database connection
+
+<div class="mb-config-catalog">
+    <section>
+        <div class="mb-config-options">
+            <div class="superfences-tabs">
+            
+            <input name="7" type="checkbox" id="_tab_7">
+                <label class="tab-selector" for="_tab_7"><i class="icon fa fa-code"></i></label>
+                <div class="superfences-content">
+                    <div class="mb-config-example">
+<pre><code class="toml">[[datasource]]
+id = "WSO2_CARBON_DB"
+url= "jdbc:h2:./repository/database/WSO2CARBON_DB;DB_CLOSE_ON_EXIT=FALSE;LOCK_TIMEOUT=60000"
+username="username"
+password="password"
+driver="org.h2.Driver"
+pool_options.maxActive=50
+pool_options.maxWait = 60000
+pool_options.testOnBorrow = true
+</code></pre>
+                    </div>
+                </div>
+                <div class="doc-wrapper">
+                    <div class="mb-config">
```

Comment:
mb-config, mb-config-options, mb-config-example etc.? is it should be mi-config
if html classes are not used for anything that's fine, but for perfectness shall we rename mb with mi ?

I guess here mb means product MB. If it means something else then fine